src/modules/fileManager/s3client.ts

```diff
@@ -0,0 +1,60 @@
+import { Client } from 'minio'
+
+import logger from './logger'
+
+export const config = {
+  bucket: process.env.MINIO_BUCKET_NAME ?? 'panda-files',
+  endPoint: process.env.MINIO_ENDPOINT ?? 'localhost',
+  port: Number.parseInt(process.env.MINIO_PORT ?? '9000'),
+  accessKey: process.env.MINIO_ACCESS_KEY ?? '12345678',
+  secretKey: process.env.MINIO_SECRET_KEY ?? '12345678',
+  useSSL: process.env.MINIO_USE_SSL?.toLowerCase() === 'true'
+}
+
+const { bucket, accessKey, secretKey, port, useSSL, endPoint } = config
+
+logger.info(
+  `S3 Config - Bucket: ${bucket} | AccessKey: ${accessKey ? '*****' : 'undefined'} | SecretKey: ${
+    secretKey ? '*****' : 'undefined'
+  } | Port: ${port} | UseSSL: ${useSSL} | EndPoint: ${endPoint}`
+)
+
+const s3Client = new Client(config)
+
+//Bucket Policy allows public read-only access
+const bucketPolicy = {
```

Comment:
@svachaj In the meantime we'll control access through the gateway but it's not a great idea in the long run as you might want to use an external S3 server in the future. This policy will need to be updated with proper OIDC configuration then.

Reply:
@svachaj I have removed the bucket policy and created a "download" route instead.